ProgressBar/themes/bootstrap/variables.less

```diff
@@ -1,2 +1,3 @@
-@transition-params: 0.3s linear 0s;
-@line-height: 1.6em;
\ No newline at end of file
+@height: 1.8em; // Controls the height of the widget (180% of the font-size)
```

Comment:
Not sure if it's dangerous or not, probably not, but there is definitely something to do on LESS consistency across widgets.
There are variables such as @star-icon-size, others like @d-switch-w or too short names like @height.

Where are defined these LESS variables is also not consistent:
- In a variables.less for the Switch in `Switch/variables.less` which is common to all themes.
- in the theme LESS file for others

About naming, I propose to follow the pattern of @d-switch-xxx for all widgets.
About where to define LESS variables, having a variables.less for each widget could be interesting in the perspective of a theming tool, to facilitate a kind of automatic introspection mechanism of all LESS variables. But I guess it's not the time to do that and we should remove the only variables.less of Switch.

I would prefer to refactor all LESS variables in a unique commit after merging all CSS related PRs.